packages/app/src/app/pages/Sandbox/Editor/Workspace/elements.js

```diff
@@ -60,6 +60,16 @@ export const getContainerStyles = props => {
     }
   `;
 
+  if (props.active) {
+    styles += `
+      color: ${getSelectedColor(
+        theme['list.activeSelectionForeground']
+      )} !important;
+      border-color: ${color()} !important;
+      background-color: ${color.lighten(0.1).clearer(0.8)()} !important;
+    `;
+  }
+
   if (props.editing) {
```

Comment:
if the file is active and you try to rename it, editing styles are overridden by active style